packages/amplify-provider-awscloudformation/src/index.ts

```diff
@@ -162,4 +167,5 @@ module.exports = {
   updateEnv,
   uploadHooksDirectory,
   transformResourceWithOverrides,
+  nestedStackFileName,
```

Comment:
Why export twice? I think we're exporting it above as well. Can we export in only place in this file please?

Reply:
this wasnt working for me when importing it , that's why I exported it twice, 
will check in next commit for this